projects/kit/components/input-phone-international/input-phone-international.style.less

```diff
@@ -0,0 +1,60 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+@flag-width: 1.75rem;
+@flag-height: 1.25rem;
+
+:host {
+    display: block;
+
+    &._disabled {
+        pointer-events: none;
+    }
+}
+
+.t-country-select {
+    width: 3.625rem;
+
+    & select:not(:disabled):not(._readonly) {
+        cursor: pointer;
+    }
+
+    &[data-size='m'] {
+        width: 5.5rem;
+    }
+
+    &[data-size='s'] {
+        width: 2rem;
+
+        .t-flag {
+            margin-left: -@space * 4;
```

Comment:
Let's ditch this variable

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/7580/commits/0b2f448326e9c9cf5a33becb91f000f7fba50525